test/benchmarks/micro/host_adler32.wast

```diff
@@ -0,0 +1,24 @@
+(module
+  (import "env" "adler32" (func (param i32 i32) (result i32)))
+  (memory (export "memory") 1)
+  (data (i32.const 0) "11223344")
```

Comment:
Should fix this to use hexadecimal notation, so we actually run it on 0x11223344 and not "1122".